tests/commons/glob/test_glob.py

```
@@ -0,0 +1,22 @@
+from sema.commons.glob import (
+    GlobMatchVisitor,
+    getMatchingGlobPaths,
+    pathMatchesGlob,
+    visitGlobPaths,
+)
+
+
+def test_getMatchingGlobPaths():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_getMatchingGlobPaths`

Add return type annotation: `None`

(ANN201)